app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_name = command.split()
+    if len(file_name) == 3:
+        if file_name[0] == "cp" and file_name[1] != file_name[2]:
+            with (
+                open(file_name, "r") as file_in,
```

Comment:
You are passing the list `file_name` as a first argument to `open`, pass the string with the required filename instead.